src/frontend/components/Header/Header.css

```diff
@@ -10,13 +10,16 @@
     padding: 10px;
   }
   
-  .navbar-div {
+  .join-container {
```

Comment:
This name is a bit confusing.
I see you want to rename the container that holds the `sign-up` `log-in`, but we need to find another semantic name for it.